sound/soc/codecs/rt1308-sdw.c

```diff
@@ -30,36 +30,13 @@
 static bool rt1308_readable_register(struct device *dev, unsigned int reg)
 {
 	switch (reg) {
-	case 0x0020:
-	case 0x0030:
-	case 0x0049:
-	case 0x0060:
-	case 0x0070:
 	case 0x00e0:
 	case 0x00f0:
-	case 0x0000 ... 0x0005:
-	case 0x0022 ... 0x0026:
-	case 0x0032 ... 0x0036:
-	case 0x0040 ... 0x0046:
-	case 0x0050 ... 0x0055:
-	case 0x0100 ... 0x0105:
-	case 0x0120 ... 0x0127:
-	case 0x0130 ... 0x0137:
-	case 0x0200 ... 0x0205:
-	case 0x0220 ... 0x0227:
-	case 0x0230 ... 0x0237:
-	case 0x0f00 ... 0x0f05:
-	case 0x0f30 ... 0x0f37:
-	case 0x2000 ... 0x200e:
-	case 0x2012 ... 0x2016:
-	case 0x201a ... 0x2027:
-	case 0x2029 ... 0x202a:
-	case 0x202d ... 0x2034:
-	case 0x2100 ... 0x213f:
-	case 0x2200 ... 0x2204:
-	case 0x2206 ... 0x2212:
-	case 0x2220 ... 0x2223:
-	case 0x2230 ... 0x2231:
+	case 0x2f01 ... 0x2f07:
```

Comment:
are the 0xe0 and 0xf0 intentional?

Reply:
@plbossart The driver will access 0xe0/0xf0 registers in rtXXX_clock_config function. So, I keep it in readable_register function.

Reply:
ok, thanks for confirming.